src/OpenTelemetry.Exporter.Jaeger/JaegerTraceExporter.cs

```diff
@@ -34,6 +34,13 @@ public class JaegerTraceExporter : SpanExporter, IDisposable
         private readonly IJaegerUdpBatcher jaegerAgentUdpBatcher;
         private bool disposedValue = false; // To detect redundant dispose calls
 
+        public JaegerTraceExporter(JaegerExporterOptions options)
```

Comment:
can you do `this(options, new JaegerUdpBatcher(options))` so we'll not need to change both ctors if we decide to change something?